settings.php

```diff
@@ -72,6 +72,19 @@
     1
 ));
 
+$fields = $DB->get_records('facetoface_session_field', [], 'name', 'id, name');
+$options = array(0 => 'None');
+foreach ($fields as $field) {
+    $options[$field->id] = $field->name;
+}
+$settings->add(new admin_setting_configselect(
+    'facetoface/column',
```

Comment:
It feels like `column` is not self explanatory enough name. May be change it to something like `displaycustomfield`?